src/server/scripts/Northrend/Ulduar/Ulduar/boss_thorim.cpp

```diff
@@ -905,9 +906,15 @@ class boss_thorim_sif : public CreatureScript
                     events.ScheduleEvent(EVENT_SIF_BLIZZARD, 15000);
                     break;
                 case EVENT_SIF_FROSTBOLT_VALLEY:
-                    me->CastSpell(me, SPELL_FROSTBOLT_VALLEY, false);
+                    _allowCast = false;
+                    events.ScheduleEvent(EVENT_SIF_STAGE_TWO, 1500);
                     events.RepeatEvent(13000);
                     return;
+                case EVENT_SIF_STAGE_TWO:
+                    me->CastSpell(me, SPELL_FROSTBOLT_VALLEY, false);
```

Comment:
Can you change the variable name to volley instead? Just for consistency 😄

Reply:
it's written valley lol, it has a typo by default, right?

Reply:
Yes, must have been someone coding till 5 am 😪